src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1517,7 +1517,7 @@ internal enum ErrorCode
         ERR_TypeReserved = 8336,
         ERR_RefExtensionMustBeValueTypeOrConstrainedToOne = 8337,
         ERR_InExtensionMustBeValueType = 8338,
-        ERR_BadParameterModifiersOrder = 8339,
+        // ERR_BadParameterModifiersOrder = 8339,
```

Comment:
>// ERR_BadParameterModifiersOrder = 8339, [](start = 8, length = 41)

I would expect the error message to be removed from the .resx file

Reply:
I would also add a small comment to explain why it was removed.

---
In reply to: [156157236](https://github.com/dotnet/roslyn/pull/23643#discussion_r156157236) [](ancestors = 156157236)